firefox-ios/Client/Coordinators/Launch/LaunchCoordinator.swift

```diff
@@ -383,3 +415,28 @@ final class LaunchCoordinator: BaseCoordinator,
         parentCoordinator?.didFinishLaunch(from: self)
     }
 }
+
+// MARK: - OnboardingServiceDelegate
+extension LaunchCoordinator: OnboardingServiceDelegate {
```

Comment:
We decided a while ago to only use extensions in rare cases. Could you please make `LaunchCoordinator` conform to these delegates directly like we did for `ParentCoordinatorDelegate` for example?